custom_components/tahoma/translations/en.json

```diff
@@ -1,22 +1,22 @@
 {
-    "config": {
-        "abort": {
-            "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
-        },
-        "error": {
-            "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
-            "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
-            "unknown": "[%key:common::config_flow::error::unknown%]"
-        },
-        "step": {
-            "user": {
-                "data": {
-                    "username": "Username",
-                    "password": "Password"
-                },
-                "description": "Enter the credentials you use for the Tahoma app or the tahomalink.com platform."   
-            }
+  "title": "Tahoma",
```

Comment:
Could you remove the `title` from all files? I understood this is not mandatory and will be filled by the integration name (Somfy TaHoma). Could you try this?

Reply:
title isn't necessary indeed. Deduced title is "Somfy Tahoma" which is good.
It seems we can't use common keys in French (and I suspect other non english languages?)
![image](https://user-images.githubusercontent.com/5701372/84145553-61094f00-aa5a-11ea-9e3e-8637407b9499.png)

Reply:
I am not sure how the common keys part works to be honest... You can translate them for now.

Reply:
did it [here](https://github.com/iMicknl/ha-tahoma/pull/45/commits/942085efcd78a47f1e2815888261953fbaad8b56)